GomokuClient/packages/gomoku-core/src/features/Board/Board.tsx

```diff
@@ -101,7 +100,7 @@ export interface BoardProps {
   tiles: TileColor[][];
   lastTile?: SwaggerTypes.TileDto;
   style?: CSSProperties;
-  winningSequence?: Pick<TileProps, "winningSequence">["winningSequence"];
+  winningSequence: SwaggerTypes.GetGameHistoryResponse["winningSequence"];
```

Comment:
Why it's not optional parametor?

Reply:
In GetGameHistory type is already nullable. I think this question mark here is just redundant. Can restore if this helps you somehow from readability point of view

Reply:
Got it, restored question mark for property